pkg/controller/datavolume/controller-base.go

```diff
@@ -1130,3 +1155,15 @@ func (r *ReconcilerBase) storageClassCSIDriverExists(storageClassName *string) (
 
 	return true, nil
 }
+
+func (r *ReconcilerBase) checkPVCUsingPopulators(pvc *corev1.PersistentVolumeClaim) (bool, error) {
+	if pvc.Spec.DataSourceRef == nil {
+		return false, nil
+	}
+	return r.storageClassCSIDriverExists(pvc.Spec.StorageClassName)
+}
+
+func (r *ReconcilerBase) usePopulatorsForPopulation(syncState *dvSyncState) (bool, error) {
```

Comment:
Maybe change this function name to something like `shouldUseCDIPopulator`, `shouldUsePopulator`, or `checkPopulatorRequirements`? I think usePopulatorsForPopulation can sound odd.

Reply:
Agree